commit 5ebf29943b291110a816a418cb9d311d6172ac1e (refs/changes/53/2253/3)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2017-08-11T20:47:42+00:00 (2 years, 2 months ago)
    
    HEAD-2364 dhcpd zone runs aground in default RSS cap
    Reviewed by: Trent Mick <trentm@gmail.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/config/sapi/services/dhcpd/service.json b/config/sapi/services/dhcpd/service.json
index f236d234..0e9a300b 100644
--- a/config/sapi/services/dhcpd/service.json
+++ b/config/sapi/services/dhcpd/service.json
@@ -2,7 +2,7 @@
     "name": "dhcpd",
     "params": {
         "archive_on_delete": true,
-        "package_name": "sdc_128",
+        "package_name": "sdc_512",
         "image_uuid": "IMAGE_UUID",
         "maintain_resolvers": true,
         "networks": [ "admin" ],
